spec.bs

```diff
@@ -622,6 +624,37 @@ members:
 The [=payment method additional data type=] for this payment method is
 {{SecurePaymentConfirmationRequest}}.
 
+### Checking if Secure Payment Confirmation is available ### {#sctn-secure-payment-confirmation-available-api}
+
+A static API is added to {{PaymentRequest}} in order to provide developers a
+simplified method of checking whether Secure Payment Confirmation is available.
+
+<xmp class="idl">
+partial interface PaymentRequest {
+    static Promise<boolean> isSecurePaymentConfirmationAvailable();
+};
+</xmp>
+<dl dfn-type="attribute" dfn-for="PaymentRequest">
+    :   {{PaymentRequest/isSecurePaymentConfirmationAvailable()}}
+    ::  Upon invocation, a promise is returned that resolves with a value of
+        `true` if the user agent believes that the Secure Payment Confirmation
```

Comment:
Is there a case where the UA would believe it is supported but it is not supported? If not, can we say "if the Secure Payment Confirmation feature is available" ?

Reply:
Removing 'believe' SGTM. If we look at ConditionalUI:

https://w3c.github.io/webauthn/#dom-publickeycredential-isconditionalmediationavailable

```
[PublicKeyCredential](https://w3c.github.io/webauthn/#publickeycredential) overrides this method to indicate availability for [conditional](https://w3c.github.io/webappsec-credential-management/#dom-credentialmediationrequirement-conditional) mediation. [WebAuthn Relying Parties](https://w3c.github.io/webauthn/#webauthn-relying-party) SHOULD verify availability before attempting to set options.[mediation](https://w3c.github.io/webappsec-credential-management/#dom-credentialrequestoptions-mediation) to [conditional](https://w3c.github.io/webappsec-credential-management/#dom-credentialmediationrequirement-conditional).

Upon invocation, a promise is returned that resolves with a value of true if [conditional](https://w3c.github.io/webappsec-credential-management/#dom-credentialmediationrequirement-conditional) [user mediation](https://w3c.github.io/webappsec-credential-management/#user-mediated) is available, or false otherwise.

This method has no arguments and returns a promise to a Boolean value.
```

So they just say 'is available'.